draft-ietf-httpbis-priority.md

```diff
@@ -634,6 +634,34 @@ signal the intended priority to the client by including the Priority field in
 a PUSH_PROMISE or HEADERS frame.
 
 
+# Retransmission Scheduling
+
+Transport protocols such as TCP and QUIC provide reliability by detecting packet
+losses and retransmitting lost information. While this document specifies
+HTTP-layer prioritization, its effectiveness can be further enhanced if the
+transport layer factors priority into scheduling both new data and
+retransmission data. The remainder of this section discusses considerations when
+using QUIC.
+
+{{!I-D.ietf-quic-transport}}, Section 13.3 states "Endpoints SHOULD prioritize
+retransmission of data over sending new data, unless priorities specified by the
+application indicate otherwise". When an HTTP/3 application uses the priority
+scheme defined in this document, the QUIC transport implementation SHOULD
+consider the relative priority of streams when scheduling of new data and
+retransmission data. For example, if one stream with higher urgency has new data
+to send and another stream with lower urgency has retransmission data to send, a
+QUIC transport is more likely to fulfil the application's intent if it chooses
+to schedule the higher urgency new data before the lower urgency retransmission
+data. While there is no requirement on QUIC implementations to implement the
+extensible priority scheme, an aligned prioritization model can help
+applications manage scheduling expectations.
```

Comment:
```suggestion
application indicate otherwise". When an HTTP/3 application uses the priority
scheme defined in this document and the QUIC transport implementation supports
application indicated stream priority, a transport that considers the relative
priority of streams when scheduling both new data and retransmission data might
better match the expectations of the application. However, here are no
requirements on how a transport chooses to schedule based on this information
because the decision depends on several factors and trade-offs, It could
prioritize new data for a higher urgency stream over retransmission data for a
lower priority stream, or it could prioritize retransmission data over new data
irrespective of urgencies.
```

As per the thread with @rmarx, this suggestion removes any formal requirement., Instead, it highlights that expectation matching might be desirable but that tradeoffs might prevent that in practicce (very much the ethos of this spec).

@yangchi if your're happy with this suggestion I think the PR is good to go. But if you disagree then we can discuss further.

Reply:
Ye i'm OK with this change. I think now that SHOULD is removed, and a "might" is used, the part after "However, here are..." isn't necessary. But i think that's just editorial bits.